lib/web3-contracts.js

```diff
@@ -277,17 +277,33 @@ export function useClaimOrder() {
   )
 }
 
-export function useTransactionReceipt() {
+export function useClaimOrderReceiptAmount() {
   const { ethersProvider } = useWalletAugmented()
 
   return useCallback(
     async hash => {
+      const abi = getKnownAbi('TOKEN_ANT')
+      const abiInterface = new EthersUtils.Interface(abi)
+
+      let amount
+
       try {
         const transactionReceipt = await ethersProvider.getTransactionReceipt(
           hash
         )
 
-        return transactionReceipt
+        transactionReceipt.logs.forEach(log => {
+          const parsedLog = abiInterface.parseLog(log)
```

Comment:
Just wanted to make sure:

- What happens if that specific log can't be parsed? Is an error thrown or is `parsedLog` just null?
- Here, since we control the previous transaction, we can just assume that the first log is going to be the `Transfer` event (I believe it is, anyway—there should only be two logs in these claim transactions).

Reply:
To answer the first question I look into Ethers code and the library return null: https://github.com/ethers-io/ethers.js/blob/master/utils/interface.js#L388

Reply:
Cool—can we double check what happens with the v5 ABI encoder?

Reply:
Thanks for looking @0xGabi , I'll log the encoder as an item to check.